.travis.yml

```diff
@@ -7,20 +7,18 @@ before_install:
 - docker push karrot96/todo-app
 - docker build --target test --tag my-test-image --cache-from karrot96/todo-app .
 script:
-- docker run -e DB_USER -e PASSWORD -e MONGO_URL -e DEFAULT_DATABASE -e WRITER_ID=19879648 my-test-image
+- docker run -e MONGO_URL -e APP_NAME my-test-image
```

Comment:
By running all of your tests in one command are you relying on the e2e tests running before the integration tests? Because if your integration tests load .env.test before the e2e tests run, they won't be able to access the real MONGO_URL.

If so, that's not ideal. If your unit tests started loading .env.test, or if you renamed a folder, your tests could break. Better to split this up so you have a separate docker run command that runs just the e2e tests, passing in `-e MONGO_URL`.